src/store.js

```diff
@@ -81,6 +85,14 @@ export default new Vuex.Store({
       context.commit('UPDATE_RULE', rule);
       this.dispatch('updateBackend');
     },
+    /*
+     * Returns current page info from the API
+     */
+    getCurrentPage: () => {
+      return api.settings.getCurrentPage().then(pageInfo => {
+        this.dispatch('PAGEINFO', pageInfo);
```

Comment:
Tu as confondu `this.dispatch` (pour une action) et `context.commit` (pour une mutation)